core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -992,15 +991,22 @@ private void listStatusInternal(LockedInodePath currInodePath,
       }
       DescendantType nextDescendantType = (descendantType == DescendantType.ALL)
           ? DescendantType.ALL : DescendantType.NONE;
+
+      String [] parentChildPathComp = null;
```

Comment:
add a comment on what this is for.

NIT: I thinks `childComponents` is a better name.

Reply:
done